build-tools/packages/build-cli/src/commands/prep/typetests.ts

```diff
@@ -0,0 +1,240 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { CliUx, Flags } from "@oclif/core";
+import chalk from "chalk";
+
+import { PreviousVersionStyle, getAndUpdatePackageDetails } from "@fluidframework/build-tools";
+
+import { BaseCommand } from "../../base";
+import { releaseGroupFlag } from "../../flags";
+
+export default class PrepareTypeTestsCommand extends BaseCommand<
```

Comment:
TODO: rename prep to prepare and add [aliases](https://oclif.io/docs/aliases) for `prep` and `gen`.